index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav>
+      <ul></ul>
+      <ul></ul>
+      <ul></ul>
+      <ul></ul>
```

Comment:
@Verde9 did you mean to have a `<ul>` and a bunch `<li>` inside
Also what are our menu items called?

Reply:
@jonly03 Just updated index.html, please review.